app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+                raise OutdatedVaccineError("Out dated vaccine")
+
+            if not visitor["wearing_a_mask"]:

```

Comment:
```suggestion
            if not visitor.get("wearing_a_mask"):

```
Use get because wearing_a_mask may not exist